csrc/aio/py_lib/py_ds_aio.cpp

```diff
@@ -27,7 +27,7 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m)
              "queue_depth"_a = 128,
              "single_submit"_a = false,
              "overlap_events"_a = false,
-             "num_threads"_a = 1)
+             "intra_op_parallelism"_a = 1)
```

Comment:
Let's also change the API `get_thread_count()` -> `get_intra_op_parallelism()`

Reply:
Also need to update the tutorial 
https://github.com/microsoft/DeepSpeed/blob/master/docs/_tutorials/deepnvme.md#parallel-file-write